apps/src/templates/instructions/TopInstructionsHeader.jsx

```diff
@@ -0,0 +1,246 @@
+import $ from 'jquery';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import PaneHeader, {PaneButton} from '@cdo/apps/templates/PaneHeader';
+import InstructionsTab from '@cdo/apps/templates/instructions/InstructionsTab';
+import CollapserIcon from '@cdo/apps/templates/CollapserIcon';
+import InlineAudio from '@cdo/apps/templates/instructions/InlineAudio';
+import {TabType} from './TopInstructions';
+import i18n from '@cdo/locale';
+import color from '../../util/color';
+import styleConstants from '../../styleConstants';
+
+const styles = {
+  paneHeaderOverride: {
+    color: color.default_text
+  },
+  audioStyleRTL: {
+    wrapper: {
+      float: 'left'
+    },
+    button: {
```

Comment:
there are a lot of duplicated styles in here. if you add `audioStyle.RTLWrapper` you can remove `audioStyleRTL` entirely. i see a lot more duplication that can be removed but you get the idea.